docs/source/release_notes.rst

```diff
@@ -14,6 +14,7 @@ Future Release
         * Add validation to Age, AgeFractional, and AgeNullable logical types (:pr:`1289`)
     * Fixes
         * Check range length in table stats without producing overflow error (:pr:`1287`)
+        * Fixes issue with initializing Woodwork Series with LatLong values (:pr:`1299`)
```

Comment:
Add your username to the list of contributors below.

Reply:
done